libs/application/template-api-modules/src/lib/modules/templates/health-insurance/health-insurance.module.ts

```diff
@@ -11,24 +11,30 @@ import { BaseTemplateAPIModuleConfig } from '../../../types'
 // Here you import your module service
 import { HealthInsuranceService } from './health-insurance.service'
 
-const XROAD_HEALTH_INSURANCE_WSDL_URL = process.env.healthInsurancewsdlUrl ?? ''
-const XROAD_HEALTH_INSURANCE_BASE_URL =
-  process.env.baseUrl ?? 'http://localhost:8080'
-const XROAD_HEALTH_INSURANCE_USERNAME =
-  process.env.healthInsuranceUsername ?? ''
-const XROAD_HEALTH_INSURANCE_PASSWORD =
-  process.env.healthInsurancePassword ?? ''
+const HEALTH_INSURANCE_XROAD_WSDLURL =
+  process.env.HEALTH_INSURANCE_XROAD_WSDLURL ??
+  'https://test-huld.sjukra.is/islandrg?wsdl'
+const HEALTH_INSURANCE_XROAD_BASEURL =
+  process.env.XROAD_BASE_PATH_WITH_ENV ?? 'http://localhost:8080'
+const HEALTH_INSURANCE_XROAD_USERNAME =
+  process.env.HEALTH_INSURANCE_XROAD_USERNAME ?? ''
+const HEALTH_INSURANCE_XROAD_PASSWORD =
+  process.env.HEALTH_INSURANCE_XROAD_PASSWORD ?? ''
+const HEALTH_INSURANCE_XROAD_CLIENT_ID = process.env.XROAD_CLIENT_ID ?? ''
+const HEALTH_INSURANCE_XROAD_ID = process.env.XROAD_HEALTH_INSURANCE_ID ?? ''
 
 export class HealthInsuranceModule {
   static register(config: BaseTemplateAPIModuleConfig): DynamicModule {
```

Comment:
You have all the variables above in `config`, right?

Reply:
yes, we added them yesterday in helm

Reply:
I mean that you don't need 
```
const HEALTH_INSURANCE_XROAD_WSDLURL =
  process.env.HEALTH_INSURANCE_XROAD_WSDLURL ??
  'https://test-huld.sjukra.is/islandrg?wsdl'
const HEALTH_INSURANCE_XROAD_BASEURL =
  process.env.XROAD_BASE_PATH_WITH_ENV ?? 'http://localhost:8080'
const HEALTH_INSURANCE_XROAD_USERNAME =
  process.env.HEALTH_INSURANCE_XROAD_USERNAME ?? ''
const HEALTH_INSURANCE_XROAD_PASSWORD =
  process.env.HEALTH_INSURANCE_XROAD_PASSWORD ?? ''
const HEALTH_INSURANCE_XROAD_CLIENT_ID = process.env.XROAD_CLIENT_ID ?? ''
const HEALTH_INSURANCE_XROAD_ID = process.env.XROAD_HEALTH_INSURANCE_ID ?? ''
```

Because it should be available in the `config` parameter.